templates/.gitignore

```diff
@@ -0,0 +1,7 @@
+## AsciiBinder-specific ignores
+_preview
+_package
+*.swp
```

Comment:
Not sure why this is needed in the templates directory.  If you make the root .gitignore file more relaxed it will cover *all* subdirectories.